lib/unexpectedMitm.js

```diff
@@ -688,10 +688,10 @@ module.exports = {
                              * hook final write callback to record raw data and immediately
                              * assert request so it occurs prior to request completion
                              */
-                             observeResponse(res).then(function (rawBuffer) {
-                                 var mockResponse = createMockResponse(rawBuffer);
-                                 assertMockResponse(mockResponse);
-                             });
+                            observeResponse(res).then(function (rawBuffer) {
+                                var mockResponse = createMockResponse(rawBuffer);
+                                assertMockResponse(mockResponse);
+                            });
 
```

Comment:
Ugh, I think there is some weird atom bug that indents the next line after a multiline comment incorrectly - seen this happen before. Totally unintentional and thanks for fixing it.